README.md

```diff
@@ -10,21 +10,26 @@ Robot State Publisher deals with two different "classes" of joint types: fixed a
+This allows another node to publish the robot description at runtime. In this mode, Robot State Publisher becomes a passive consumer of the description and no longer republishes it.
+
 Examples showing how to pass the `robot_description` parameter using a launch file are available in the 'launch' subdirectory.
 
 Published Topics
 ----------------
-* `robot_description` (`std_msgs/msg/String`) - The description of the robot URDF as a string.  Republishes the value set in the `robot_description` parameter, which is useful for getting informed of dynamic changes to the URDF.  Published using the "transient local" quality of service, so subscribers should also be "transient local".
+* `robot_description` (`std_msgs/msg/String`) - *(Only if `description_from_topic` is false (default))* The description of the robot URDF as a string. Republishes the value set in the `robot_description` parameter, which is useful for informing other tools of the current robot model. Published using the "transient local" quality of service, so subscribers should also use "transient local".
 * `tf` (`tf2_msgs/msg/TFMessage`) - The transforms corresponding to the movable joints of the robot.
 * `tf_static` (`tf2_msgs/msg/TFMessage`) - The transforms corresponding to the static joints of the robot.
 
 Subscribed Topics
 -----------------
-* `joint_states` (`sensor_msgs/msg/JointState`) - The joint state updates to the robot poses.  The RobotStatePublisher class takes these updates, does transformations (such as mimic joints), and then publishes the results on the tf2 topics.
+* `joint_states` (`sensor_msgs/msg/JointState`) - The joint state updates to the robot poses. The RobotStatePublisher class takes these updates, does transformations (such as mimic joints), and then publishes the results on the tf2 topics.
+* `robot_description` (`std_msgs/msg/String`) - *(Only if `description_from_topic` is true (not default))*. The incoming robot description from another node. The node will not use the `robot_description` parameter if this mode is enabled.
 
 Parameters
 ----------
-* `robot_description` (string) - The original description of the robot in URDF form.  This *must* be set at robot_state_publisher startup time, or the node will fail to start.  Updates to this parameter will be reflected in the `robot_description` topic.
-* `publish_frequency` (double) - The maximum frequency at which non-static transforms (e.g. joint states) will be published to `/tf`.  Defaults to 20.0 Hz.
-* `ignore_timestamp` (bool) - Whether to accept all joint states no matter what the timestamp (true), or to only publish joint state updates if they are newer than the last publish_frequency (false).  Defaults to false.
-* `frame_prefix` (string) - An arbitrary prefix to add to the published tf2 frames.  Defaults to the empty string.
+* `description_from_topic` (bool) - Whether to receive the robot description from the `robot_description` topic instead of from a parameter. Defaults to false.
```

Comment:
```suggestion
* `use_robot_description_topic` (bool) - Whether to receive the robot description from the `robot_description` topic instead of from a parameter. Defaults to false.
```